### ISSUE TITLE
without full output path DeepMedic divides output between relative output path folder and a copy in ...CapTK\bin then crashes

### ISSUE THREAD

**Issue by pefs44** [CBICA/CaPTk#270]
**Describe the bug**
when running deepmedic from command line, using just  a folder name or relative path, the normalized versions of the t1, t1gd, flair and t2 images are placed in the designated folder. At the same time however a folder with the same name is created in the bin subdirectory of the CapTK home folder. At the point of starting full segmentation however the program looks in the latter folder for the normalized images, and not finding them crashes with an error NE 0. 
Setting the full path for the output, all of the files are generated in the proper target folder and no crash occurs.
 
**To Reproduce**
1. Run DeepMedic with the output set to a folder name without path                                                                                                                                                                                                                                                                                                                                                                                                    
2. watch CapTK/bin folder with filemanager to see a folder with target name generated and populated
3. watch target with filemanager to see a folder with target name generated and populated by normalized images
4. See error in terminal (see screenshots below)

**Expected behavior**
wehn starting segmentation, DeepMedic should look in the target folder not a subdirectory of CapTK\bin.

**Screenshots**

c:\prog\CapTK1.6.1\bin\DeepMedic -t1c 3dt1gd\3dt1gd.nii -t1 3dt1\3dt1_reg_to_3dt1gd.nii -fl flair\flair_reg_to_3dt1gd.nii -t2 t2\t2_reg_to_3dt1gd.nii -o  .\dmsegm -L d:\data\pclin_glio\nii\pclin_glio_40\20160805\dmlog
command starts, 
various pages of messages then:
~~~~~~~~~~~~~~~~~~~~ Segmenting subject with index #0 ~~~~~~~~~~~~~~~~~~~~
Loading subject with 1st channel at: c:\prog\CapTK1.6.1\bin\dmsegm\t1_normalized.nii.gz
Traceback (most recent call last):
  File "deepMedicTestApi.py", line 188, in <module>
  File "deepmedic\frontEndModules\deepMedicTest.py", line 356, in deepMedicTestMain
  File "deepmedic\routines\testing.py", line 138, in performInferenceOnWholeVolumes
  File "deepmedic\dataManagement\sampling.py", line 343, in load_imgs_of_single_case
  File "deepmedic\image\io.py", line 20, in loadVolume
  File "site-packages\nibabel\loadsave.py", line 40, in load
FileNotFoundError: No such file: 'c:\prog\CapTK1.6.1\bin\dmsegm\t1_normalized.nii.gz'
[18960] Failed to execute script deepMedicTestApi
DeepMedic exited with code !=0.

**CaPTk Version**
1.6.1

**Desktop (please complete the following information):**
 Windows 10.1803]

**Comment by sarthakpati** [comment]
We will be making a host of changes to the way DM works in the next release. Thanks for reporting.

**Comment by sarthakpati** [comment]
Fixed in **1.7.0.Beta**. Reopen if issue persists.
